tests/functional_tests/t0_main/llm/t4_llm_openai_chat_completion.py

```diff
@@ -1,10 +1,9 @@
 import openai
-from wandb.integration.openai import autolog
-
-autolog()
+from wandb.integration.openai import autolog as openai_autolog
 
 
 def main():
+    openai_autolog(init=dict(project="openai_logging"))
```

Comment:
why? the idea was to test starting the autolog in multiple parts of the code

Reply:
moved back